cmd/commands/walletrpc_active.go

```diff
@@ -268,9 +268,10 @@ var bumpFeeCommand = cli.Command{
 		cli.Uint64Flag{
 			Name: "conf_target",
 			Usage: `
-	The deadline in number of blocks that the input should be spent within.
-	When not set, for new inputs, the default value (1008) is used; for
-	exiting inputs, their current values will be retained.`,
+	The conf target is the starting fee rate of the fee function expressed
```

Comment:
We should also update the docs for `bumpclosefee`. In addition, we should add the new param `deadline_delta` in the cli too.

Reply:
very good point, updated the bumpclosefee and bumpforceclose RPCs as well.

Reply:
also added the conf_target (target_conf) to the bumpforceclose RPC